support-frontend/assets/pages/weekly-subscription-checkout/components/weeklyCheckoutForm.jsx

```diff
@@ -334,6 +334,7 @@ function WeeklyCheckoutForm(props: PropTypes) {
             title="Your card details"
           >
             <StripeProviderForCountry
+              key={props.country} // this is necessary to force the component to use the correct Stripe account, see https://reactjs.org/docs/lists-and-keys.html#keys
```

Comment:
rather than trying to remember it everywhere, could we just put it in one place here(see below link), and base it on the actual stripe public key rather than the country?
https://github.com/guardian/support-frontend/blob/950743bb04089deec52051ac233484d5d6a8209a/support-frontend/assets/components/subscriptionCheckouts/stripeForm/stripeProviderForCountry.jsx#L21

Reply:
I thought I'd tried it at that level without success, but trying it again it appears to work so 👍